lib/upipe-ts/libupipe_ts.pc.in

```diff
@@ -5,6 +5,6 @@ includedir=@includedir@
 Name: libupipe_ts
 Description: Upipe multimedia framework, MPEG-2 transport stream modules
 Version: @VERSION@
-Requires: libupipe libupipe_modules
+Requires: libupipe libupipe_modules @PKGCONFIG_REQUIRES@
```

Comment:
I believe this should go to `Requires.private` instead (see `upipe-dvb` or `upipe-ebur128`). Then `pkg-config --static --libs` should be used if static linking is required. Otherwise `libtasn1` and `libgcrypt` would be linked to dynamically linked programs, which is not wanted. Also, `PKGCONFIG_REQUIRES` might be too generic as this is specific to `upipe-ts`.